### DISCUSSION TITLE
notice viewer install problem

### DISCUSSION THREAD

**Discussion by dansto** [OP-TED/eForms-SDK#15] [Q&A] [answered]
hi, mabe someone from the community can help me with this install problem:
https://github.com/OP-TED/eforms-notice-viewer/issues/7

**Comment by rousso** [comment, marked as answer]
Hi @dansto,
The notice viewer has not been released yet.
We are still sorting out some issues that were caused by the changes we made while moving the project from our internal Git repositories to Github. We are working on that and we will do a release of the notice viewer sample app as soon as we can.

We will reply to the issue that you created in the notice viewer repository.

By the way, I "accidentally" enabled the Github discussions yesterday for the eForms SDK. 
We are not ready for that either. We will properly launch discussions asap.

Thanks
Yannis
